### PR TITLE
[DET-2017] Open CFP, update logo, sponsor maintenance.

### PR DESCRIPTION
**Pull request by broamski**

### CODE REVIEW

File: data/events/2017-detroit.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ location: "A. Alfred Taubman Center for Design Education, College for Creative S
 nav_elements: # List of pages you want to show up in the navigation of your page.
  
 #  - name: program
-#  - name: propose
-#    url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+  - name: propose
+    # url: https://devopsdaysdetroit.typeform.com/to/HfQ847
```
Comment (yvovandoorn) [inline data/events/2017-detroit.yml]: Hey @broamski, is this intentionally still commented out?

Comment (broamski) [inline data/events/2017-detroit.yml]: Yes. I didn't prefer the behavior where it overrode the `PROPOSE` url, so I commented it out in the event we'd switch back later on.

Comment (yvovandoorn) [inline data/events/2017-detroit.yml]: 👍

**Comment by broamski** [comment]
Thanks!

**Pull request merged**